samples/boards/ehl_crb/gpio/sample.yaml

```diff
@@ -0,0 +1,7 @@
+sample:
+  description: Example of using GPIOs on Intel Elkhart Lake board
+  name: Example of using GPIOs on Intel Elkhart Lake board
```

Comment:
Would it make sense to change these to be platform agnostic (e.g. just say "Intel boards")?

Reply:
In that case, would we need to change the location of the app as well since it will be no longer specific to one board??

Reply:
I suppose so, but I really don't want to make this a blocker for this PR, so I'm fine keeping this EHL-specific for now.